api/views/session_views.py

```diff
@@ -1,7 +1,15 @@
 from django.db import transaction
 from rest_framework.response import Response
 from rest_framework.views import APIView
-from api.models import *
+from api.models import (
+    Quiz,
+    QuizSession,
+    QuizSessionStudent,
+    UserResponse,
+    QuestionMultipleChoice,
+    InstructorRecordings,
+    LectureSummary,
+)
```

Comment:
Was this flagged by `flake8`?

Reply:
No, but the star import was causing a lot of warnings in my editor. I don't know if PyCharm uses a different linter/language server than what I'm using, so maybe that was only an issue for me. I believe avoiding star imports is generally considered a best practice.